src/Components/test/E2ETest/Tests/InputLargeTextAreaTest.cs

```diff
@@ -0,0 +1,208 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using BasicTestApp;
+using BasicTestApp.FormsTest;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.E2ETesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.Tests
+{
+    public class InputLargeTextAreaTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>>
+    {
+        private static readonly string[] CircuitErrors = new[]
+        {
+            "Connection disconnected with error 'Error: Server returned an error on close: Connection closed with an error.'.",
+            "Cannot send data if the connection is not in the 'Connected' State.",
+            "HubConnection.connectionClosed(Error: Server returned an error on close: Connection closed with an error.) called while in state Connected.",
+        };
+
+        public InputLargeTextAreaTest(
+            BrowserFixture browserFixture,
+            ToggleExecutionModeServerFixture<Program> serverFixture,
+            ITestOutputHelper output)
+            : base(browserFixture, serverFixture, output)
+        {
+        }
+
+        protected override void InitializeAsyncCore()
+        {
+            Navigate(ServerPathBase, noReload: _serverFixture.ExecutionMode == ExecutionMode.Client);
+            Browser.MountTestComponent<InputLargeTextAreaComponent>();
+        }
+
+        [Fact]
+        public void CanSetValue()
+        {
+            // Set large value
+            var setTextBtn = Browser.Exists(By.Id("setTextBtn"));
+            setTextBtn.Click();
+
+            var valueInTextArea = GetTextAreaValueFromBrowser();
+            Assert.Equal(new string('c', 50_000), valueInTextArea);
+
+            FocusAway();
+            AssertLogDoesNotContainMessages(CircuitErrors);
```

Comment:
This is kinda super crummy way to test this - if the text of our log messages change, and now start appearing, this would continue to succeed. Is there a better way to detect this? For e.g. can we check for something else that ensures that Blazor Server is still operating?

Reply:
Ah that's a great point, thanks! Updated to do a basic interactivity check by incrementing a counter.